x-pack/test/functional/apps/lens/group6/legacy_metric.ts

```diff
@@ -34,7 +34,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       });
     });
 
-    it('should color the metric text based on value', async () => {
+    it.skip('should color the metric text based on value', async () => {
```

Comment:
This may pass again once we revert to `style`
```suggestion
    it('should color the metric text based on value', async () => {
```